Android/Samples/libraries/geojson/src/main/java/com/microsoft/maps/geojson/GeoJsonParser.java

```diff
@@ -138,6 +141,38 @@ private JSONArray getCoordinates(@NonNull JSONObject object) throws JSONExceptio
     return object.getJSONArray("coordinates");
   }
 
+  private void createIconAndAddToLayer(
+      @NonNull Geoposition position,
+      @NonNull AltitudeReferenceSystemWrapper altitudeReferenceSystemWrapper) {
+    MapIcon icon = mFactory.createMapIcon();
+    icon.setLocation(
+        new Geopoint(position, altitudeReferenceSystemWrapper.getAltitudeReferenceSystem()));
+    mLayer.getElements().add(icon);
+  }
+
+  private void createLineAndAddToLayer(
+      @NonNull ArrayList<Geoposition> positions,
+      @NonNull AltitudeReferenceSystemWrapper altitudeReferenceSystemWrapper) {
+    MapPolyline line = mFactory.createMapPolyline();
+    line.setPath(
+        new Geopath(positions, altitudeReferenceSystemWrapper.getAltitudeReferenceSystem()));
+    mLayer.getElements().add(line);
+  }
+
+  private void createPolygonAndAddToLayer(
+      @NonNull ArrayList<ArrayList<Geoposition>> positionLists,
+      @NonNull AltitudeReferenceSystemWrapper altitudeReferenceSystemWrapper) {
```

Comment:
The wrapper isn't used in this function, just the altitude reference system. Pass just the altitude reference system.